acquia_lift.module

```diff
@@ -65,6 +65,10 @@ function acquia_lift_page_attachments(array &$page) {
   $page_context = \Drupal::service('acquia_lift.service.context.page_context');
   $page_context->setPageContextTitle($title);
   $drupal_settings['pageContext'] = $page_context->getAll();
+  $metatags = $page_context->getMetatags();
+  $metatags_lift = $page_context->getLiftMetatags($credential_settings);
```

Comment:
https://github.com/acquia/acquia_lift/blob/LEX-484/src/Service/Context/PageContext.php#L80

PageContext already has $settings, we can just get it from there (handle like "field_mappings").